README.md

```diff
@@ -1,3 +1,88 @@
-# TODO Repository Name
+# Knative Monitoring
 
-_TODO(leads): please fill in this section_
+This repository contains Grafana Dashboards and Prometheus Scraping configurations to monitor Knative.
+
+You will need to deploy the following ServiceMonitors.
+
+Also, kube-state-metrics v2+ doesn't export all labels by default which means you must pass the following argument to kube-state-metrics:
+
+```
+- --metric-labels-allowlist=pods=[*],deployments=[app.kubernetes.io/name,app.kubernetes.io/component,app.kubernetes.io/instance]
+```
+Once kube-state-metrics has been adjusted, you will need to deploy the following servicemonitors and import the dashboards in the grafana folder.
+
+```
```

Comment:
Should we move these ServiceMonitors to a file?

Reply:
I never liked the doing `k apply -f https://foo.abr/potato.yaml `. I prefer copying and pasting yamls to a file if <200 lines

Reply:
sure - we can revisit this later